cmd/server.go

```diff
@@ -174,6 +175,10 @@ var stringFlags = []stringFlag{
 			" Only set if using TFE as a backend." +
 			" Should be specified via the ATLANTIS_TFE_TOKEN environment variable for security.",
 	},
+	{
+		name:        SlackTokenFlag,
+		description: "API token required for Slack webhook integration",
```

Comment:
What kind of API token is required? I forget. 

I wouldn't say "slack webhook" because it's a slack notification, not a webhook.

Reply:
This is a "bot" token, and I've reworded it. Probably worth putting in an another PR for #42 though.